source/puppet/4.4/reference/config_important_settings.markdown

```diff
@@ -171,6 +172,8 @@ Puppet Server has [its own configuration files][puppetserver_config_files]; cons
 * [`jruby-puppet`][jruby_puppet] --- Provides details on tuning JRuby for better performance.
 * [`JAVA_ARGS`][jvm_heap_config] --- Instructions on tuning the Puppet Server memory allocation.
 
+Also, Puppet Server uses its own [auth.conf][] file by default instead of its `puppet-admin` section to authorize client requests.
```

Comment:
I think you should just replace that first bullet point instead of writing an addendum — change it to link to the Puppet Server auth.conf page, and say that you have to add auth rules in order to use the admin interface (and maybe say the admin interface is most useful for refreshing environments). 

Less confusing?